example-config.yml

```diff
@@ -409,3 +409,7 @@ dateTime:
 #   # Whether to render routes within flex zones of a route's patterns. If set to true,
 #   # routes will not be rendered within flex zones.
 #   hideRouteShapesWithinFlexZones: true
+
+# API key to make Mapillary API calls. These are used to show street imagery.
+# mapillary:
+#   key: <Mapillary Key>
```

Comment:
I'm not seeing an API key provided from Mapillary on the dev portal. They provide a client token and a client secret, at least with the way I registered the app. What do I put in this field? Can we call this something else? Or clarify in the comment?

Reply:
I've made a clearer comment in f68f92513ae1214fa5cea5e9cdd287fcd36cc78a!